packages/app/src/runner/SpecRunner.vue

```diff
@@ -3,68 +3,84 @@
     id="main-pane"
     class="flex border-gray-900 border-l-1"
   >
-    <HideDuringScreenshot
-      id="inline-spec-list"
-      class="bg-gray-1000"
+    <ResizablePanels
+      :offset-left="64"
+      :max-total-width="windowWidth - 64"
+      :initial-panel1-width="initialSpecsListWidth"
+      :initial-panel2-width="initialReporterWidth"
+      :show-panel1="runnerUiStore.isSpecsListOpen && !screenshotStore.isScreenshotting"
```

Comment:
Eg, if we did change the panels name, this would be

```
:show-spec-list="showSpecList"

showSpecList = computed(() => runnerUiStore.isSpecsListOpen && !screenshotStore.isScreenshotting)
```

what do you think? Any particular reason to go with panelX instead of more explicit names?

Reply:
I think it helps when reading/thinking about the ResizablePanels component to have the names be more generic, just to be clear that the logic around resizing doesn't care that specific content is being used in the slot. 

I'd be up for different names though - the difference just being a number, and the number sometimes being an internal part of a variable name, can make it a little easy to misread. Almost went for more explicit `leftPanel`, `middlePanel`, `rightPanel`.